src/components/swiper/index.js

```diff
@@ -176,10 +177,10 @@ const Swiper = ( props ) => {
 	const renderNavigation = useMemo( () => {
 		return (
 			<>
-				<button className={ `nav-button__prev ${ navigation === false && 'no-navigation' }` } id={ `${ uuid }-prev` }>
+				<button className={ navigationClass ? `${ navigationClass }__prev` : `nav-button__prev ${ navigation === false && 'no-navigation' }` } id={ `${ uuid }-prev` }>
 					<svg className="icon" style={ { transform: 'rotate(180deg)' } } />
 				</button>
-				<button className={ `nav-button__next ${ navigation === false && 'no-navigation' }` } id={ `${ uuid }-next` }>
+				<button className={ navigationClass ? `${ navigationClass }__next` : `nav-button__next ${ navigation === false && 'no-navigation' }` } id={ `${ uuid }-next` }>
```

Comment:
This is a little difficult to read now that we are nesting a tertiary as well here. How do you feel about moving this to use the `classnames` package? For the sake of posterity, I think it would help with understanding the conditional.
```javascript
/**
 * External dependencies
 */
import classnames from 'classnames';

const buttonClasses = classnames( {
			[ `${ navigationClass }__next` ]: navigationClass,
			'nav-button__next': ! navigationClass,
			'no-navigation': navigation === false
		} );
```

Reply:
updated this in the newest commit